OpenRA.Mods.Common/Traits/Buildings/Refinery.cs

```diff
@@ -44,6 +44,10 @@ public class RefineryInfo : TraitInfo, Requires<WithSpriteBodyInfo>, IAcceptReso
 		[Desc("Discard resources once silo capacity has been reached.")]
 		public readonly bool DiscardExcessResources = false;
 
+		[GrantedConditionReference]
+		[Desc("Condition to grant while being docked.")]
```

Comment:
```suggestion
		[Desc("Condition to grant while a harvester is docked.")]
```

Reply:
This name doesn't hold up long term. it's not going to be just harvester in the near future

Reply:
`while an actor is docked`?

Reply:
sure